main.js

```diff
@@ -0,0 +1,63 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const books = [];
+
+const AddBook = (book) => {
+  if (localStorage.getItem('books') === null) {
+    localStorage.setItem('books', JSON.stringify([]));
+  }
+  books.push(book);
+  const booksData = JSON.stringify(books);
+  localStorage.setItem('books', booksData);
+};
+
+const displayBooksList = () => {
+  const listSection = document.getElementById('list');
+
+  const tag = document.querySelector('.list');
+  let child = tag.lastElementChild;
+  while (child) {
+    tag.removeChild(child);
+    child = tag.lastElementChild;
+  }
+  const booksData = JSON.parse(localStorage.getItem('books'));
+  // eslint-disable-next-line no-unused-vars
+  booksData.map((book, index) => {
+    const oneBook = document.createElement('div');
+    oneBook.innerHTML = `
+    <span>"${book.title}"</span>
+    <span>"${book.author}"</span>
+    <button type="button" onclick="remove(${index})">Remove</button>`;
+
+    listSection.appendChild(oneBook);
+    return oneBook;
+  });
+};
```

Comment:
- Good job done so far, but according to the project requirements, all previous functionality of the app should be maintained, when I reload the app, it doesn't display books that were previously in the local storage. Please fix this 🙏
Your project:
<img width="848" alt="rtrthh" src="https://github.com/BilalLiaquat7/sneak_peak/assets/73607512/57baa0a1-24f2-4a01-8c81-c7fd1ff552cf">